frontend/js/utils/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const baseUrl = window.location.host;
+
+const apiUrls = {
+  battleDetail: (id) => `api/battles/${id}`,
+};
+
+const getFromApi = (urlApi) => {
+  const url = `http://${baseUrl}/${urlApi}`;
```

Comment:
We are using `django-js-reverse` to pass urls from django to react. Instructions on how to set this here: https://github.com/ierror/django-js-reverse#use-the-urls-in-javascript